app/components/project-card-list.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component, PropTypes } from 'react';
+import ProjectCard from '../partials/project-card';
+
+class ProjectCardList extends Component {
+  componentDidMount() {
+    document.documentElement.classList.add('on-secondary-page');
+  }
+
+  componentWillUnmount() {
+    document.documentElement.classList.remove('on-secondary-page');
+  }
+
+  render() {
+    return (
+      <div className="project-card-list">
+        {this.props.projects.map(project => (
+          <div key={project.id}>
```

Comment:
Why is this in a `div` when the `key` can be applied directly to the `ProjectCard`?